test/crawler/keyword/keywords_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Crawler.Keyword.KeywordsTest do
+  use Crawler.DataCase, async: true
+
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.Keyword
+
+  describe "list_keywords/1" do
+    test "given a valid user id, returns user's keyword list" do
+      %{id: user_id} = insert(:user)
+      insert(:keyword, user_id: user_id)
+
+      %{id: expected_user_id} = insert(:user)
+      expected_keyword = insert(:keyword, user_id: expected_user_id)
+
+      assert Keywords.list_keywords(expected_user_id) == [expected_keyword]
+    end
+  end
+
+  describe "create_keyword/1" do
+    test "given a valid keyword data, creates a keyword" do
```

Comment:
do we need to test the `invalid keyword data` case here 🤔

Reply:
Updated in [1d816b2](https://github.com/vnntsu/elixir-ic/pull/36/commits/1d816b234f96b49e84b553a585e8119e4990ed31)